imageorsnap.yaml

```diff
@@ -0,0 +1,257 @@
+            then
+              curl -L -O https://dl.influxdata.com/telegraf/releases/telegraf_1.10.3-1_amd64.deb
+              dpkg -i telegraf_1.10.3-1_amd64.deb
+              # Snoopy dependency install
+              if [ $commandLog = True ]
+              then
+                apt-get update -y
+                apt-get install libtool-bin -y
+              fi
+            elif [ $bootImage = "Centos-7.5" ] || [ $bootImage = "RHEL-7.5" ]
+            then
+              curl -L -O https://dl.influxdata.com/telegraf/releases/telegraf-1.10.3-1.x86_64.rpm
+              yum localinstall telegraf-1.10.3-1.x86_64.rpm -y
+              # Snoopy dependency install
+              if [ $commandLog = True ]
+              then
+                yum update -y
+                yum install libtool -y
+                sed -i -e 's/secure/auth.log/' /etc/rsyslog.conf
+                service rsyslog restart
+              fi
+            else
+              echo "Unknown image, assuming debian OS"
+              curl -L -O https://dl.influxdata.com/telegraf/releases/telegraf_1.10.3-1_amd64.deb
+              dpkg -i telegraf_1.10.3-1_amd64.deb
+              # Snoopy dependency install
+              if [ $commandLog = True ]
+              then
+                apt-get update -y
+                apt-get install libtool-bin -y
+              fi
+            fi
+
+            ################snoopy install
+            if [ $commandLog = True ]
+            then
+              rm -f snoopy-install.sh
+              curl -L -o snoopy-install.sh https://github.com/a2o/snoopy/raw/install/doc/install/bin/snoopy-install.sh
+              chmod 755 snoopy-install.sh
+              ./snoopy-install.sh git-master
+              chmod -R 755 /var/log/auth.log
+              snoopy-enable
+            else
+              echo "snoopy not installed"
+            fi
+
+            ################# Telegraf agent config
+            curl -L -o jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+            chmod +x ./jq
+            mv jq /usr/bin
+            echo "serverid=$(sudo curl GET  -s 'http://169.254.169.254/openstack/latest/meta_data.json' |jq -r '.uuid' )" >> /etc/default/telegraf
+            mv /etc/telegraf/telegraf.conf /etc/telegraf/telegraf-orig.conf
+            if [ $isprodvm = "non-prod" ]
+            then
+              curl -o /etc/telegraf/telegraf.conf https://raw.githubusercontent.com/mayankkapoor/statuscheck/master/telegraf_nonprod.conf
+            else
+              curl -o /etc/telegraf/telegraf.conf https://raw.githubusercontent.com/mayankkapoor/statuscheck/master/telegraf_prod.conf
+            fi
+            curl -o /etc/telegraf/longrunning.sh https://raw.githubusercontent.com/mayankkapoor/statuscheck/master/longrunning.sh
+            curl -o /etc/telegraf/commands.sh https://raw.githubusercontent.com/mayankkapoor/statuscheck/master/commands.sh
```

Comment:
Change statuscheck branchname to `test`, i.e., replace statuscheck/master/ with statuscheck/test/